src/admin-views/aggregator/settings.php

```diff
@@ -575,8 +634,32 @@
 			'name'     => __( 'Other URLs', 'the-events-calendar' ),
 			'priority' => 45,
 		),
+		'eventbrite-settings' => array(
+			'name'     => __( 'Eventbrite', 'the-events-calendar' ),
+			'priority' => 17,
+		),
 	);
 
+	/**
+	 * If Eventbrite Tickets is enabled and Event Aggregator is disabled, display the correct import links
+	 *
+	 * @since TBD
+	 */
+	if ( class_exists( 'Tribe__Events__Tickets__Eventbrite__Main' ) && ! tribe( 'events-aggregator.main' )->has_license_key() ) {
+		$ea_keys = array(
+			'ical-settings',
```

Comment:
Why are we unset everything?

Reply:
Hey @bordoni , thanks for the review :+1: ! This unset exists to remove only the import setting links related to Event Aggregator when it is disabled (the user doesn't have a license key) and Eventbrite Tickets is enabled.

Reply:
So this code probably needs to live in Eventbrite Tickets. Right? If it only comes into play if you have it active.

Reply:
It only comes to play if both things are happening simultaneously: EA is disabled + EBT is enabled.
What makes things more complex here is the fact that Eventbrite tickets is enabling EA by default (without a license key). This is my understanding:

Eventbrite import settings should be displayed by default on EA (because it is possible to import from EB without Eventbrite Tickets)
If the user enables EBT, nothing happens, the import settings are the same
If the user doesn't have a license key for EA, all import settings are removed
If the user doesn't have a license key for EA, but has EBT enabled, all import settings should be removed, except Eventbrite
I believe it makes more sense to leave the code on EA because in this case, we are still disabling the default EA import settings if the user doesn't have a license key and just adding an extra check in case of EBT is enabled (i.e. don't remove the default EA settings for Eventbrite if EBT is active).